editor/animation_bezier_editor.cpp

```diff
@@ -216,8 +216,7 @@ void AnimationBezierTrackEdit::_notification(int p_what) {
 
 		case NOTIFICATION_ENTER_TREE: {
 			panner->setup((ViewPanner::ControlScheme)EDITOR_GET("editors/panning/animation_editors_panning_scheme").operator int(), ED_GET_SHORTCUT("canvas_item_editor/pan_view"), bool(EditorSettings::get_singleton()->get("editors/panning/simple_panning")));
-			[[fallthrough]];
-		}
+		} break;
```

Comment:
Nitpick, there should be a separation line between `case`s (unless it's a fallthrough) for readability, as they're often quite big.

Reply:
It'd be neat if this could be added to clang-format